src/app/hooks/queries/ordinals/useAddressRareSats.ts

```diff
@@ -67,15 +67,31 @@ export const useGetUtxoOrdinalBundle = (
   });
 
   const bundle = data?.txid ? mapRareSatsAPIResponseToBundle(data) : undefined;
-  const inscriptionRange = bundle?.satRanges.find((range) =>
-    range.inscriptions.some((inscription) => inscription.inscription_number === ordinalNumber),
-  );
+
+  if (ordinalNumber) {
```

Comment:
not sure about this change, but ive noticed that we need this because when on `confirmOrdinalTransaction` screen, need to return data correctly when detecting to send an ordinal, or a satBundle